### DISCUSSION TITLE
Search history gone

### DISCUSSION THREAD

**Discussion by SpicySpaceman** [community/community#64200] [Mobile]
The search history (recent searches below the search input) randomly gets purged. Unfortunately I am not sure what triggers this, but it keeps happening all the time which is quite annoying. Hopefully I'm not the only one with this issue, I couldn't find anything related here though.  

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.126.0 (107057637)
Device: iPhone15,2 (iOS 16.6)
</details>

**Comment by rainbow1016** [comment]
very good question!

**Comment by Manuel36791** [comment]
I think it’s a bug in the GitHub cite every now and then I face the same problem
